main.go

```diff
@@ -170,6 +170,20 @@ func getBasicInfoOfType(c echo.Context) error {
 	var agencies []strModels.Agency
 	var err error
 	var estadual bool
+	jurisdicao := map[string]string{
+		"justica-eleitoral":    "Eleitoral",
+		"ministerios-publicos": "Ministério",
+		"justica-estadual":     "Estadual",
+		"justica-do-trabalho":  "Trabalho",
+		"justica-federal":      "Federal",
+		"justica-militar":      "Militar",
+		"justica-superior":     "Superior",
+		"conselhos-de-justica": "Conselho",
+	}
+	// Adaptando as URLs do site com o banco de dados
+	if jurisdicao[groupName] != "" {
+		groupName = jurisdicao[groupName]
```

Comment:
Não seria melhor pensar em uma resposta específica para quando o grupo não for encontrado? Por exemplo, caso o usuário digite `dadosjusbr.org/meu-grupo-querido`, qual seria o melhor retorno?

Minha sugestão é que essa API retorne um not found (código 404).